app/models/facility.rb

```diff
@@ -101,8 +102,20 @@ def to_s
     "#{name} (#{abbreviation})"
   end
 
+  def problem_order_details
```

Comment:
I went back and forth on this. On the front end, there are "problem orders" and "problem reservations" and I went with similar naming here. But `reservations` and `non_reservations` make more sense, internally.